src/Home/Home.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { NewBookForm } from '../NewBookForm/NewBookForm';
+import { BookContainer } from '../BookContainer/BookContainer';
+
+const data = [
+    {title: 'The Hungry Caterpillar', pages: 35},
+    {title: 'Harry Potter', pages: 400}
```

Comment:
Once we hook up with the work on the Back-end, we'll need a `useEffect()` here in which we fetch the user's existing reads

@ninesky00 @catherinemdean15